HISTORY.md

```diff
@@ -1,5 +1,30 @@
 # History
 
+## 1.8.0 - 2023-10-30
+
```

Comment:
Can you add a summary of the main changes? Here I think that would be the addition of the 'random' missing value replacement strategy, the `UniformEncoder` and `ClusterBasedNormalizer` bugs and the randomization bug. Can we also call out the deprecation?